DOCS/man/input.rst

```diff
@@ -1893,10 +1893,8 @@ Hooks
 
 Hooks are synchronous events between player core and a script or similar. This
 applies to client API (including the Lua scripting interface). Normally,
-events are supposed to be asynchronous, and the hook API provides an awkward
-and obscure way to handle events that require stricter coordination. There are
-no API stability guarantees made. Not following the protocol exactly can make
-the player freeze randomly. Basically, nobody should use this API.
+events are supposed to be asynchronous, and the hook API provides a way to
+handle events that require stricter coordination.
```

Comment:
> Not following the protocol exactly can make the player freeze randomly.

Should be left intact. The hook api may be used, but is still dangerous tool in hands on scripts, and should be used with caution.

`Not following the protocol exactly can make the player freeze randomly. Use with caution, avoid if possible.`

Reply:
Apparently in the C api you have to call `mpv_hook_continue` but this is done automatically in scripts. But I think "randomly" makes it sound like it will occasionally break unpredictably. If you don't continue mpv will freeze, it is not random.